aten/src/ATen/cuda/Atomic.cuh

```diff
@@ -6,6 +6,10 @@
 
 #include <ATen/NumericUtils.h>
 
+#if !(defined(USE_ROCM) || ((defined(CUDA_VERSION) && CUDA_VERSION < 11000) || (defined(__CUDA_ARCH__) && (__CUDA_ARCH__ < 800))))
+#include <cuda_bf16.h>
```

Comment:
I'm not sure why this is needed, if it it removed, the build fails with the complaint that `__nv_bfloat16` isn't defined when the included `c10/util/BFloat16.h` header should also include it...